### ISSUE TITLE
1.4.6 Contrast (Enhanced) (Level AAA)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#539]
#### [1.4.6 Contrast (Enhanced)](https://www.w3.org/TR/WCAG22/#contrast-enhanced)

(Level AAA)

>The visual presentation of [text](https://www.w3.org/TR/WCAG22/#dfn-text) and [images of text](https://www.w3.org/TR/WCAG22/#dfn-images-of-text) has a [contrast ratio](https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio) of at least 7:1, except for the following:
>
><dl><dt>Large Text</dt>
><dd><a href=https://www.w3.org/TR/WCAG22/#dfn-large-scale>Large-scale</a> text and images of large-scale text have a contrast ratio of at least 4.5:1;</dd>
><dt>Incidental</dt>
><dd>Text or images of text that are part of an inactive <a href=https://www.w3.org/TR/WCAG22/#dfn-user-interface-components>user interface component</a>, that are <a href=https://www.w3.org/TR/WCAG22/#dfn-pure-decoration>pure decoration</a>, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.</dd>
><dt>Logotypes</dt>
><dd>Text that is part of a logo or brand name has no contrast requirement.</dd></dl>

##### Applying SC 1.4.6 Contrast (Enhanced) to Non-Web Documents and Software

This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.4.6](https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced.html#intent).

***
**For WCAG2ICT consideration:** Doesn't seem that any additional notes are needed. Comment on the issue if something should be added.
***
